packages/react/src/components/header/components/headerActionBar/HeaderActionBar.tsx

```diff
@@ -155,7 +155,7 @@ export const HeaderActionBar = ({
         </div>
       </div>
       <HeaderLanguageSelector fullWidthForMobile />
-      <HeaderActionBarNavigationMenu />
+      <HeaderActionBarNavigationMenu logo={<span className={styles.logoWrapper}>{logo}</span>} logoProps={logoProps} />
     </>
```

Comment:
There is quite a lot of repetition with logos

Line: 137:
```
 <LinkItem {...logoProps}>
    <span className={styles.logoWrapper}>{logo}</span>
  </LinkItem>
```

Then in HeaderActionBarNavigationMenu.tsx

```
<LinkItem {...logoProps} className={styles.logoLink}>
        {logo}
      </LinkItem>
```
where the `{logo}` is actually a `<span className={styles.logoWrapper}>{logo}</span>`

Could these be merged by making a <LogoLinkItem /> component which handles both? :)

Reply:
https://github.com/City-of-Helsinki/helsinki-design-system/pull/1122/commits/9f34c1e9d7bce8ed4fb958153715d095fa7e5494